packages/core/src/html/includePanelProcessor.ts

```diff
@@ -230,6 +230,24 @@ export function processInclude(node: MbNode, context: Context, pageSources: Page
     const $ = cheerio.load(actualContent);
     const actualContentOrNull = $(hash).html();
     actualContent = actualContentOrNull || '';
+    if (actualContent !== '') {
+      const footnotePattern = /<a aria-describedby="footnote-label" href="#fn-\d+-\d+">\[\d+\]/g;
+      const matches = actualContent.match(footnotePattern);
+      if (matches) {
+        const hrefPattern = /href="#(fn-\d+-\d+)"/;
+        let footnoteNumber = 1;
+        let toAppend = '<mb-temp-footnotes>';
+        matches.forEach((match) => {
+          const href = match.match(hrefPattern)![1];
+          const replaceTo = `<a aria-describedby="footnote-label" href="#${href}">[${footnoteNumber}]`;
```

Comment:
Why is it necessary to do this replace here? Why are you changing the number for the footnote?

Reply:
I have improved this with the following changes:
<img width="768" alt="Screenshot 2024-03-14 at 16 54 05" src="https://github.com/MarkBind/markbind/assets/121547057/b8587d55-411f-4ec9-9248-ea9977a33ca2">

where the default value is: 
<img width="285" alt="Screenshot 2024-03-14 at 16 54 00" src="https://github.com/MarkBind/markbind/assets/121547057/18249748-469f-4c95-b840-db48146eb4ea">

last modification: 
<img width="730" alt="Screenshot 2024-03-14 at 16 54 13" src="https://github.com/MarkBind/markbind/assets/121547057/5c77430f-ed11-407e-be3a-d4077ca1627e">

Reply:
Motivation being, as this initially is to fix the error in display footnote in the user doc, I decided to be defensive and make sure that at least the footnote from a single hash import is going to look good (while not go beyond the scope and do not make changes that is outside the` hash` part)